src/components/UserProfile/TeamsAndProjects/UserProjectsTable.jsx

```diff
@@ -65,6 +65,7 @@ const UserProjectsTable = React.memo(props => {
 
   useEffect(() => {
     setFilteredTasks(() => filterTasksByUserTaskSituation(actualType));
+    console.log('userTasks:', userTasks);
```

Comment:
Debugging should be removed here as well

Reply:
@kamtoeddy thanks. Please review again!